content/docs/dvclive/index.md

```diff
@@ -1,18 +1,25 @@
 # dvclive
 
-[`dvclive`](/doc/dvclive) is an open-source Python library for monitoring the
-progress of metrics during training of machine learning models.
+[`dvclive`](https://github.com/iterative/dvclive) is an open-source Python
```

Comment:
Need to decide whether to use `dvclive` or DVCLive (see the last bullet in this doc) throughout docs and readme.

Reply:
I have no strong preference. For know and in other to be consistent along the docs, I used DVCLive when referring to the "tool" and `dvclive` when referring to the "API module"; not sure it that makes sense